### PR TITLE
[14.0][MIG] fieldservice_account_payment

### PR DESCRIPTION
**Pull request by Freni-OSI**

### CODE REVIEW

**Comment by smangukiya** [comment]
@Freni-OSI @AmmarOfficewalaSerpentCS  Can you please execute pre-commit, and fix travis so that we can clean it up? As dependency is clear for this module.

CC: @brian10048

**Comment by Freni-OSI** [comment]
Please review
CC: @smangukiya @patrickrwilson

**Review by patrickrwilson — APPROVED** [review]
Code looks good. LGTM

**Review by smangukiya — APPROVED** [review]

File: fieldservice_account_payment/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "fieldservice_account",
+    ],
+    "data": [
+        "security/ir.model.access.csv",
+        "views/fsm_order.xml",
+        "views/account_payment.xml",
+    ],
+    "license": "AGPL-3",
+    "development_status": "Production/Stable",
+    "maintainers": ["max3903"],
```
Comment (smangukiya) [inline fieldservice_account_payment/__manifest__.py:21]:
```suggestion
    "maintainers": ["max3903", "smangukiya"],
```

**Pull request merged**